credentials/customer-loyalty/credential.json

```
@@ -0,0 +1,25 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://contexts.vcplayground.org/examples/customer-loyalty/v1.json"
+  ],
+  "type": ["VerifiableCredential", "CustomerLoyaltyCredential"],
+  "description": "Earn Big Rewards at Big Retail Store by presenting your loyalty card!",
+  "image": "https://playground.chapi.io/examples/customer-loyalty/image.png",
```

Use vcplayground.org:
```suggestion
  "image": "https://vcplayground.org/examples/customer-loyalty/image.png",
```